### PR TITLE
Update to latest

### PR DESCRIPTION
**Pull request by geeknoid**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/extensions/pull/4031)

### CODE REVIEW

**Comment by RussKie** [comment]
@geeknoid the dependencies must be available at one of the managed feeds (typically dotnet8). [Here's the procedure](https://github.com/dotnet/arcade/blob/main/Documentation/MirroringPackages.md) to mirror those from nuget.org.

File: eng/Packages/General.props [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <PackageVersion Include="Microsoft.WindowsAzure.ServiceRuntime" Version="3.0.1382.30" />
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageVersion Include="OpenTelemetry.Api" Version="1.4.0" />
-    <PackageVersion Include="OpenTelemetry.Exporter.Geneva" Version="1.4.0" />
+    <PackageVersion Include="OpenTelemetry.Exporter.Geneva" Version="1.4.1" />
     <PackageVersion Include="OpenTelemetry.Extensions.Hosting" Version="1.4.0" />
```
Comment (RussKie) [inline eng/Packages/General.props:31]: Any reason not to align the versions?

**Comment by RussKie** [comment]
I requested the new packages to be mirrored to dotnet-public.

**Comment by geeknoid** [comment]
@RussKie Thanks for getting the package in the feed. I don't have permission to do this, so I had asked Jose to get it, but his PR failed for some reason (I don't have permissions to look at the PR, so I don't know why...).

As for why not to align the versions of the package, I'm just mirroring what's in the ADO repo.

**Pull request merged**